### PR TITLE
New ft:db App

### PR DESCRIPTION
**Pull request by oxoovo**
New ft:db app. It works with the 1.0.0.

### CODE REVIEW

File: packages/ftdb/ftdb.py [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+
+class PicDialog(TouchDialog):
+    def __init__(self, parent, img_name):
+        TouchDialog.__init__(self, "Picture", parent)
+        
+        self.vbox = QVBoxLayout()
+        
+        self.ctx = ssl.create_default_context()
+        self.ctx.check_hostname = False
+        self.ctx.verify_mode = ssl.CERT_NONE
```
Comment (hypnotoad) [inline packages/ftdb/ftdb.py:34]: Can you check if it works without this hack?

**Review by hypnotoad — APPROVED** [review]
It looks great! Would propose to adjust the font sizes (too big in my opinion), but that could be done later.

File: packages/ftdb/ftdb.py (hunk 1 above)
Comment (oxoovo) [reply packages/ftdb/ftdb.py:34]: No, I get this error: `urllib.error.URLError: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1129)>`

Comment (hypnotoad) [reply packages/ftdb/ftdb.py:34]: FTC will resolve this issue when they move to another server.

**Comment by oxoovo** [comment]
> It looks great! Would propose to adjust the font sizes (too big in my opinion), but that could be done later.

I used a smaller font for the title and smaller icons in the list.

**Review by hypnotoad — APPROVED** [review]

**Comment by harbaum** [comment]
It's really great so see you guys reviving the CFW!!!!

File: packages/ftdb/ftdb.py (hunk 1 above)
Comment (ski7777) [reply packages/ftdb/ftdb.py:34]: I did not run the app yet. Is there a way to disable cert check as long as we know that the server is broken?

Comment (hypnotoad) [reply packages/ftdb/ftdb.py:34]: We can remove the hack afterwards. I already checked with the appstore that ignoring ssl errors is not needed any more.

**Comment by oxoovo** [comment]
Could somebody merge it?

**Pull request merged**